dimension/extraction.go

```diff
@@ -37,7 +42,8 @@ func (e *MissingDimensionValues) Error() string {
 }
 
 // New returns a new Extract object for a given instance
-func New(dimensions map[string]string, dimensionColumnOffset int, headerRow []string, datasetAPIURL string, datasetAPIAuthToken string, instanceID string, line []string, maxRetries int, timeColumn int) *Extract {
+func New(dimensions map[string]string, dimensionColumnOffset int, headerRow []string, datasetAPIURL string, datasetAPIAuthToken string,
```

Comment:
Dont need a constructor to set exported fields on an exported struct with no logic

Reply:
I removed this function